composer/loggers/mlflow_logger.py

```diff
@@ -62,7 +67,8 @@ def init(self, state: State, logger: Logger) -> None:
             self.run_name += f'-rank{dist.get_global_rank()}'
```

Comment:
Can we check for the `MLFLOW_RUN_ID` [env variable](https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html?highlight=environment%20variables#mlflow.environment_variables.MLFLOW_RUN_ID) as well? Should a similar change to line 63